test/type_prop/normalize.cpp

```diff
@@ -20,48 +20,6 @@
 
 using namespace std;
 using namespace ngraph;
-TEST(type_prop, normalize_invalid_input_tensor_rank)
-{
-    Shape data_shape{1, 2, 3, 4, 5};
-    auto data = make_shared<op::Parameter>(element::f32, data_shape);
-    auto axes = make_shared<op::Parameter>(element::u64, Shape{1, 2});
-    float eps{1e-6f};
-    auto eps_mode = op::EpsMode::ADD;
-
-    try
-    {
-        auto normalize = make_shared<op::NormalizeL2>(data, axes, eps, eps_mode);
-        // Should have thrown, so fail if it didn't
-        FAIL() << "Invalid input tensor rank.";
-    }
-    catch (const NodeValidationFailure& error)
-    {
-        EXPECT_HAS_SUBSTRING(error.what(),
-                             std::string("Input tensor rank must be 2, 3 or 4 dimensional"));
-    }
-    catch (...)
-    {
-        FAIL() << "Deduced type check failed for unexpected reason";
-    }
-
-    data = make_shared<op::Parameter>(element::f32, Shape{2});
-
-    try
-    {
-        auto normalize = make_shared<op::NormalizeL2>(data, axes, eps, eps_mode);
-        // Should have thrown, so fail if it didn't
-        FAIL() << "Invalid input tensor rank.";
-    }
-    catch (const NodeValidationFailure& error)
-    {
-        EXPECT_HAS_SUBSTRING(error.what(),
-                             std::string("Input tensor rank must be 2, 3 or 4 dimensional"));
-    }
-    catch (...)
-    {
-        FAIL() << "Deduced type check failed for unexpected reason";
-    }
-}
 
```

Comment:
Could you please explain how this test works:
```
Shape data_shape{2, 3, 4};
auto data = make_shared<op::Parameter>(element::f32, data_shape);
const auto axes = make_shared<op::Constant>(element::u64, Shape{3}, vector<int64_t>{1, 2, 3});
```
How you can apply axes{1,2,3} to input_shape{2,3,4}?

Reply:
I agree with with you. For me it doesn't make sense too. It is case connected with `Reshape` for `input_shape.size() == 4`. I didn't remove it in first version of this PR because I was not sure of somebody use it. Not it was removed.